### PR TITLE
return centroid of clusters and adding device for different GPU training

### PR DESCRIPTION
**Pull request by HaoKang-Timmy**
Sometimes in model compression, there is a need of centroids.
You could make it optional, please.
Also add device in init function, used for different GPU training

### CODE REVIEW

**Comment by fabiogiglietto** [comment]
both improvements seem to be very useful to me. I encourage the author to merge after testing.

**Comment by DeMoriarty** [comment]
Apologies for the delay. fast-pytorch-kmeans is intended to have a similar interface as [sklearn.cluster.KMeans](https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html#sklearn.cluster.KMeans.fit_predict), in which `fit_predict` only returns labels, and the centroids (a.k.a cluster_centers_) are accessed as an attribute of the KMeans class instance. I think this should've been mentioned in README. 
Adding support for different devices would be very useful, but I think inferring it from the input tensor in `fit()` would be more user friendly than having to explicitly specify it in `__init__` .
